src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -663,6 +663,22 @@ def getRule(self, ruleId):
             self.logger.error("Exception getting rule id: %s. Error: %s", ruleId, str(ex))
         return res
 
+    def updateRule(self, ruleId, opts):
+        """
+        Update rule information for a given rule id
+        :param ruleId: string with the rule id
+        :param opts: rule id options passed to Rucio
```

Comment:
Is it a dict? Please be specific in the docstring.

Reply:
done